src/doom/m_menu.c

```diff
@@ -1449,6 +1449,18 @@ boolean M_Responder (event_t* ev)
             key = key_menu_abort;
             joywait = I_GetTime() + 5;
         }
+
+        // Fill-in the savegame name if user press Fire on the joystick
+        if (saveStringEnter && joybmenu >= 0 && ev->data1&1)
+        {
+            // Create a savegame string
+            char savestring[SAVESTRINGSIZE];
+
+            memset(savestring, 0, SAVESTRINGSIZE);
+            M_snprintf(savestring, SAVESTRINGSIZE - 1, "JOYSTICK SLOT %i", saveSlot);
```

Comment:
I like it indicating that it was the save you did that time you were playing with a joystick. It's slightly more helpful for remembering or identifying what save is what.